TOC.md

```diff
@@ -23,7 +23,7 @@
 - Develop
   - [Overview](/develop/dev-guide-overview.md)
   - Quick Start
-    - [Build a TiDB Cluster in TiDB Cloud (Developer Tier)](/develop/dev-guide-build-cluster-in-cloud.md)
+    - [Build a TiDB Cluster in TiDB Cloud (Serverless Tier)](/develop/dev-guide-build-cluster-in-cloud.md)
```

Comment:
TODO: update TOC.md in master and other release branches.

Reply:
Just want to make sure I understand it correctly. We need to change TOC.md in all release branches and master branch, am I right?

Reply:
Got another question, if we change this in other release. Can we redirect users to serverless tier document in release-6.1 branch? Otherwise they might see document that's still using the term Developer Tier in that branch.

Reply:
Got it. And master branch is for dev.

Reply:
Just realized there is a Japanese document branch. And it used Japanese which I don't know how to translate Serverless Tier🤔

Reply:
That's OK. The Japanese branch is generated by machine translation.

Reply:
I see.